.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,98 +1,76 @@
-        `Authors`, `Editors` or `Contributors`.
-
-        > Use **Authors** if you have created and substantially added content.  
-        > Use **Editor** if you have made substantial edits or review.  
-        > Use **Contributor** if you have made minor edits, reviews or
-        > contributions.  
-        > If you've done multiple, pick the most weighted: Author > Editor >
-        > Contributor.  
-        > If you prefer to remain anonymous, that's fine too, but note that a
-        > record of your contributions based on your GitHub username will exist
-        > here in the codebase.
-
-        </details>
-
-    -   **REQUIRED** I have ensured that the
-        [project's central bibliography](https://github.com/systemphil/sphil/blob/main/README_BIBLIOGRAPHY.md)
-        contains the necessary bibliographical details for the citations I have
-        used.
-
-    -   _Optional_ My article is a stub or I want to actively encourage
-        contribution, I've added the Stub component to the bottom of my content
-        or where relevant:
-
-        ```ts
-        import Stub from "@/components/Stub";
-
-        <Stub />;
-        ```
-
--   **If Docs contribution is unchecked:** Code contribution
-    ([Apache version 2 license](https://www.apache.org/licenses/LICENSE-2.0.txt))
-      <details>
-
-    All code apart of what is inside `src/pages/**` (excluding
-    `/contributing/**`, `_app.mdx`, `_document.tsx`, `_meta.json`,
-    `acknowledgements.mdx`, `index.mdx`, `privacy.mdx`, `team.mdx`, `terms.mdx`)
-    is subject to Apache version 2 license. Basically, anything outside of
-    content, literature, philosophy.
-
-      </details>
+
+## Author's Note
+
+<!-- Briefly describe your changes and their purpose -->
+
+<details>
+<summary>📝 Guidelines for Philosophical/Literary Contributions</summary>
+
+### License Agreement
+
+By submitting this PR, you agree to license your content under the
+[CC BY-NC-SA 4.0 License](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+
+### Required Steps
+
+1. Follow the
+   [formatting guidelines](https://github.com/systemphil/sphil/blob/dev/src/pages/contributing/formatting/basic-markdown.md)
+2. Use
+   [Chicago author-date style](https://www.chicagomanualofstyle.org/tools_citationguide/citation-guide-2.html)
+   for citations
+3. Update the
+   [project's central bibliography](https://github.com/systemphil/sphil/blob/main/README_BIBLIOGRAPHY.md)
+4. Add metadata at the top of your file:
+
+```md
+---
+title: Your Title Here
+description: Brief description of your content
+isArticle: true
+authors: Your Name (Year)
+editors: Editor Name (Year)
+contributors: Contributor Name
+---
+
+## Your Article Title
+```
+
+### Optional
+
+Consider adding the Stub component to encourage further contributions.
+
+</details>
+
+<details>
+<summary>💻 Guidelines for Code Contributions</summary>
+
+### Required Steps
+
+1. Ensure code adheres to project coding standards
```

Comment:
Haha, this means we'll have to write what they are! Once things stabilize a bit, we'll update README and add a CONTRIBUTING textfile in the repo. Perhaps you could open an issue on this. :)